plcc/src/proteinstructure/Molecule.java

```diff
@@ -566,46 +574,52 @@ public Integer getAtomRadius() {
         switch(type) {
             case 0:
                 atomRadius = Settings.getInteger("plcc_I_aa_atom_radius");
+                break;
             case 1:
-                atomRadius = Settings.getInteger("plcc_I_lig_atom_radius");       // no value for "Other" so far --> Default is AA
+                atomRadius = Settings.getInteger("plcc_I_lig_atom_radius");     // no value for "Other" so far --> Default is AA
+                break;
             case 2:
                 atomRadius = Settings.getInteger("plcc_I_aa_atom_radius");
+                break;
             case 3:
                 atomRadius = Settings.getInteger("plcc_I_rna_atom_radius");
-//          default:
-//              System.err.println("Tried to get distance between Molecules. Molecule could not be matched with an atom radius.");
+                break;
+            default:
+                atomRadius = Settings.getInteger("plcc_I_aa_atom_radius");
+                DP.getInstance().e("No Radius for this Molecule. Tried to move on with default value.");
```

Comment:
If the program can move on, I would throw a warning instead of an error. Maybe I am contradicting my earlier self here ^^' In the message it should say 'Trying' as it is not done, yet. Nice that you thought of the default case.